src/_macosx.m

```diff
@@ -832,6 +851,10 @@ int mpl_check_modifier(
         {"destroy",
          (PyCFunction)FigureManager_destroy,
          METH_NOARGS},
+        {"_set_window_mode",
+         (PyCFunction)FigureManager__set_window_mode,
+         METH_VARARGS,
+         "Set the window open mode (auto, tab, window)"},
```

Comment:
```suggestion
         "Set the window open mode (system, tab, window)"},
```

Looks like one more missed `s/auto/system/`

Reply:
Oops, thanks! I just committed that via the website, so feel free to squash this down to get rid of that commit.